hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/smtype/CreateUpdateSoftwareTypeLayout.java

```diff
@@ -612,7 +612,7 @@ private void discard() {
      */
     private void crateNewSWModuleType() {
```

Comment:
There is also a typo in the method name `crateNewSWModuleType() ` => `createNewSWModuleType()`